pkgs/development/libraries/aqbanking/gwenhywfar.nix

```
@@ -10,22 +10,18 @@
   ]
 }:
 
-let
-  inherit ((import ./sources.nix).gwenhywfar) sha256 releaseId version;
-in stdenv.mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "gwenhywfar";
-  inherit version;
+  version = "4.99.22rc6";
 
-  src = let
-    qstring = "package=01&release=${releaseId}&file=02";
-    mkURLs = map (base: "${base}/sites/download/download.php?${qstring}");
-  in fetchurl {
-    name = "${pname}-${version}.tar.gz";
-    urls = mkURLs [ "http://www.aquamaniac.de" "http://www2.aquamaniac.de" ];
-    inherit sha256;
+  src = fetchFromGitHub {
+    owner = "aqbanking";
+    repo = "gwenhywfar";
+    rev = version;
+    sha256 = "0ndr2vpp4kikpg0w5fdclxgbsxqz1jygazd4ll8r2asx4jdz54lx";
   };
 
-  configureFlags = [
+    configureFlags = [
```

Nitpick: Indentation

```suggestion
  configureFlags = [
```

---

fixed